CHANGELOG.md

```diff
@@ -263,6 +263,9 @@ Adding a new version? You'll need three changes:
   to those having this label set to "true". This limits the amount of resources that are kept in memory.
   The default value is `konghq.com/configmap`.
   [#6753](https://github.com/Kong/kubernetes-ingress-controller/pull/6753)
+- Added the possibility to set `Secret`s as `CACertificateRefs` in the `BackendTLSPolicy`
+  objects.
+  [#6853](https://github.com/Kong/kubernetes-ingress-controller/pull/6853)
```

Comment:
Can we move this to the entry above which introduces support for `BackendTLSPolicy`? For the user this reading this I believe it will be cleaner to have all that in 1 point. WDYT?